.codefresh/honeydipper-sphinx.yml

```diff
@@ -0,0 +1,35 @@
+version: '1.0'
+
+steps:
+  main_clone:
+    title: Cloning honeydipper-sphinx repository...
+    type: git-clone
+    repo: honeydipper/honeydipper-sphinx
+    git: github
+    revision: master
+
+  generate_doc:
+    title: 'generate docs'
+    image: honeydipper/honeydipper:1.0.2
+    environment:
+      - DOCSRC=docgen
+      - DOCDST=source
+    commands:
+      - '/opt/honeydipper/drivers/builtin/honeydipper docgen'
+
+  git_commit:
+    title: 'git commit'
+    image: alpine/git:latest
+    commands:
+      - mkdir -p ~/.ssh
+      - echo -n "${SSH_KEY}" | base64 -d > ~/.ssh/id_rsa
```

Comment:
Is it possible to use the git provider function from codefresh instead of going this route?.

Reply:
Might also be worth just doing it in Circle?